content/guide/_index.md

```diff
@@ -8,145 +8,207 @@ Welcome! This guide will help you get started with a Hugo Theme, including how t
 port install hugo
 ```
 
 * Homebrew (Linux)
-``` bash
+```bash
 brew install hugo
 ```
 
 * Chocolatey (Windows)
-``` bash
+```bash
 choco install hugo-extended -confirm
 ```
 
 * Scoop (Windows)
-``` bash
+```bash
 scoop install hugo-extended
 ```
 
-#### Local development in a nutshell
+{{% notice note %}}For more information about other installation methods check out the [Hugo Documentation](https://gohugo.io/getting-started/installing/).{{% / notice %}}
 
-{{< youtube dyZcRRWiuuw >}}
+### Step 2: Initiate Hugo website
 
-**If the list above sounds like a foreign language to you, that's okay! Let's break it down:**
+Next we create a skeleton Hugo website.
 
- * **`Hugo` is a static site generator**. If you don't have Hugo installed, you'll need to [download](https://github.com/gohugoio/hugo/releases) and [install](https://gohugo.io/getting-started/installing/) it [make sure you have downloaded the [Hugo Extended](#install-hugo-extended-version) version for compiling scss files].
+```bash
+hugo new site my-hugo-site
+```
 
- * **A local server** is a temporary server environment on your computer-generated by your theme's build tool. This avoids browser limitations of serving files directly from your computer's filesystem (i.e. [CORS](https://developer.mozilla.org/en-US/docs/Glossary/CORS) limitations).
+### Step 3: Set up a Git repository
 
- * **Live reload** is a convenience function that many themes will include as part of their build tooling. It simply means that when you edit source files after they are recompiled, your browser will magically refresh and display the changes without a manual refresh.
+While this is not a mandatory requirement, we advise to set up a repository with your website code in it. This will make updates of your theme easier, and you have a backup of older versions of your website in case you mess up things when trying out new features.
 
-### What are the source, compiled, and static files?
+We will leave out how to create an account with Github, Bitbucket or Gitlab to create a repository online and keep on our own computer for now.
 
-These terms are can mean different things in different contexts, but for the purposes of a Hugo Theme:
+```bash
+cd my-hugo-website
+git init
+```
 
- * **Source files** are files that are meant to be processed by a theme's build tools.
+### Step 4: Adding a theme
 
- * **Compiled files** are generated as a result of running a compiling process (also called a "build process") 
-on the source files.
+There are three ways to add a theme. Choose the one most convenient for you. 
 
- * **Static files** are ones that aren't processed or generated.
+#### Method 1: Install the theme by copying
 
-**Here are a few real-world examples of each type of file:**
+If you have a compressed (.zip, .tar or any other format) file with the theme then extract it and copy the resulting folder into the folder named `themes` in your repository. You should end up with a folder structure like the following:
```

Comment:
Codacy found an issue: [[maximum-line-length] Line must be at most 80 characters](https://app.codacy.com/gh/gethugothemes/gethugothemes-docs/pullRequest?prid=6304190)